CRM/Member/Form/Membership.php

```diff
@@ -718,8 +718,8 @@ public function buildQuickForm() {
     $this->assign('isRecur', $isRecur);
 
     $this->addFormRule(array('CRM_Member_Form_Membership', 'formRule'), $this);
-
-    $this->assign('outBound_option', Civi::settings()->get('mailing_backend'));
+    $mailingInfo = Civi::settings()->get('mailing_backend');
```

Comment:
Per my comment - this line was actually broken.

Would still like you to review & extend.